paper/paper.md

```diff
@@ -348,131 +255,29 @@ In the case of collisions, the time-step adaptivity is aimed at eliminating erro
-
-```python
-from PySDM import Formulae
-from PySDM_examples.Singer_Ward.aerosol import AerosolBetaCaryophylleneDark
-aerosol = AerosolBetaCaryophylleneDark()
-
-models = {
-    Formulae(surface_tension='Constant'),
-    Formulae(
-        surface_tension='CompressedFilmOvadnevaite',
-        constants={
-            'sgm_org': 35 * si.mN / si.m,
-            'delta_min': 1.75 * si.nm
-        }
-    ),
-    Formulae(
-        surface_tension='CompressedFilmRuehl',
-        constants={
-            'RUEHL_nu_org': aerosol.modes[0]['nu_org'],
-            'RUEHL_A0': 115e-20 * si.m * si.m,
-            'RUEHL_C0': 6e-7,
-            'RUEHL_m_sigma': 0.3e17 * si.J / si.m**2,
-            'RUEHL_sgm_min': 35 * si.mN / si.m
-        }
-    ),
-    Formulae(
-        surface_tension='SzyszkowskiLangmuir',
-        constants={
-            'RUEHL_nu_org': aerosol.modes[0]['nu_org'],
-            'RUEHL_A0': 115e-20 * si.m * si.m,
-            'RUEHL_C0': 6e-7,
-            'RUEHL_sgm_min': 35 * si.mN / si.m
-        }
-    )
-}
-```
-
-Using these different models for the surface-partitioning, we can demonstrate the effect variable surface 
-  tension has on the activation of aerosol with some organic fraction.
-The presence of the orgnaics both modifies the surface tension and the hygroscopicity, resulting sometimes
-  in a Köhler curve with local minima features.
-Below is (psuedo-)code used to generate four Köhler curves for the same partially organic aerosol particle,
-  just under different assumptions of surface-partitioning by the insoluble organic species. 
-
-```python
-for model in models:
-    r_wet = np.logspace(
-        np.log(50 * si.nm),
-        np.log(2000 * si.nm),
-        base=np.e, num=100
-    )
-    sigma = np.ones(len(r_wet))
-    for j,vw in enumerate(model.trivia.volume(r_wet)):
-        sigma[j] = model.surface_tension.sigma(
-            300 * si.K,
-            vw,
-            model.trivia.volume(50 * si.nm),
-            aerosol.modes[0]['f_org']
-        )
-    RH_eq = model.hygroscopicity.RH_eq(
-        r_wet,
-        300 * si.K,
-        aerosol.modes[0]['kappa'][model.surface_tension.__name__],
-        (50 * si.nm)**3,
-        sigma
-    )
-```
-![Köhler curves for aerosol under 4 assumptions of thermodynamic surface-partitioning of organic species.](Singer_fig1_kohler.pdf){#fig:kohler width="100%"}
 
 # Author contributions
 
 EDJ led the formulation and implementation of the collisional breakup scheme with contributions from JBM.
-PB led the formulation and implementation of the adaptive time-stepping schemes for diffusional and collisional growth.
+PB led the formulation and implementation of the adaptive time-stepping.
 KD contributed to setting up continuous integration workflows for the GPU backend. 
-CES contributed the aerosol initialisation framework.
+CES contributed the aerosol initialisation framework and the .
```

Comment:
@slayoo Were you trying to edit this?

Reply:
@claresinger, sorry! completely unintended, I was just intending to shorten the sentence about adaptivity.